src/router/router.rs

```diff
@@ -196,62 +196,62 @@ fn creates_valid_regex_for_routes () {
 fn can_match_var_routes () {
     let route_store = &mut Router::<()>::new();
 
-    route_store.add_route(Method::Get, "/foo/:userid", middleware! { "hello from foo" });
-    route_store.add_route(Method::Get, "/bar", middleware! { "hello from foo" });
-    route_store.add_route(Method::Get, "/file/:format/:file", middleware! { "hello from foo" });
+    route_store.add_route(Method::GET, "/foo/:userid", middleware! { "hello from foo" });
+    route_store.add_route(Method::GET, "/bar", middleware! { "hello from foo" });
+    route_store.add_route(Method::GET, "/file/:format/:file", middleware! { "hello from foo" });
 
-    let route_result = route_store.match_route(&Method::Get, "/foo/4711").unwrap();
+    let route_result = route_store.match_route(&Method::GET, "/foo/4711").unwrap().0;
     assert_eq!(route_result.param("userid"), Some("4711"));
 
-    let route_result = route_store.match_route(&Method::Get, "/bar/4711");
+    let route_result = route_store.match_route(&Method::GET, "/bar/4711");
     assert!(route_result.is_none());
 
-    let route_result = route_store.match_route(&Method::Get, "/foo");
+    let route_result = route_store.match_route(&Method::GET, "/foo");
     assert!(route_result.is_none());
 
     // ensure that this will work with commas too
-    let route_result = route_store.match_route(&Method::Get, "/foo/123,456");
+    let route_result = route_store.match_route(&Method::GET, "/foo/123,456");
     assert!(route_result.is_some());
 
-    let route_result = route_result.unwrap();
+    let route_result = route_result.unwrap().0;
     assert_eq!(route_result.param("userid"), Some("123,456"));
 
     // ensure that this will work with spacing too
-    let route_result = route_store.match_route(&Method::Get, "/foo/John%20Doe");
+    let route_result = route_store.match_route(&Method::GET, "/foo/John%20Doe");
     assert!(route_result.is_some());
 
-    let route_result = route_result.unwrap();
+    let route_result = route_result.unwrap().0;
     assert_eq!(route_result.param("userid"), Some("John%20Doe"));
 
     // check for optional format param
-    let route_result = route_store.match_route(&Method::Get, "/foo/John%20Doe.json");
+    let route_result = route_store.match_route(&Method::GET, "/foo/John%20Doe.json");
     assert!(route_result.is_some());
 
-    let route_result = route_result.unwrap();
+    let route_result = route_result.unwrap().0;
     assert_eq!(route_result.param("userid"), Some("John%20Doe"));
     assert_eq!(route_result.param("format"), Some("json"));
 
     // ensure format works with queries
-    let route_result = route_store.match_route(&Method::Get,
+    let route_result = route_store.match_route(&Method::GET,
     "/foo/5490,1234.csv?foo=true&bar=false");
     assert!(route_result.is_some());
 
-    let route_result = route_result.unwrap();
+    let route_result = route_result.unwrap().0;
     // NOTE: `.param` doesn't cover query params currently
```

Comment:
I know this is existing code, but this comment is troubling.  It's vague and not a doc comment, so it feels like a pitfall for users to walk into.

Reply:
Cleaned up the docs, with a pointer to the `query_params` module.